awq/utils/calib_data.py

```diff
@@ -47,7 +47,7 @@ def get_calib_dataset(
             line = data[text_column]
             line = line.strip()
             line_encoded = tokenizer.encode(line)
-        if len(line_encoded) > 512:
+        if len(line_encoded) > block_size:
```

Comment:
By the way I am not sure if this check is even needed, the extra tokens could be packed into the next block.